File: TypeScript/Arrays/Sort/InsertionSort.ts
const InsertionSort = (arr: number[])=>{
    for (let i = 0; i < arr.length; i++) {
       for (let j = 0; j < i; j++) {
         if (arr[i] < arr[j]) {
             let temp: number = arr[i]
             arr[i] = arr[j]
             arr[j] = temp
         }
       }
        
    }
    console.log(arr);
    
}

let arr : number[] = [2,22,1,65,34,98,4]
InsertionSort(arr)

File: Python/Arrays/Sort/InsertionSort.py
def InsertionSort(arr: list[int])->None:
    for i in range(len(arr) -1,-1,-1):
        maximumIndex:int = i
        for j in range(0,i):
            if arr[j] > arr[maximumIndex]:
                maximumIndex = j
        temp : int = arr[i]
        arr[i] = arr[maximumIndex]
        arr[maximumIndex] = temp
    
    


arr: list[int] = [32,3,52,1,4,53,2]
InsertionSort(arr)